exercise4/removeDuplicates.js

```diff
@@ -0,0 +1,3 @@
+function removeDuplicates(array) {
+  return new Set(array);
+}
```

Comment:
- Kindly modify the return value of the as well. Currently you are returning an `object`  like this `Set(5) { 1, 2, 3, 4, 5 }` .
- Since we passed in an array it should return an array of unique values like `[ 1, 2, 3, 4, 5 ]`